app/src/main/res/layout/fragment_default_variant.xml

```diff
@@ -1,64 +1,61 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
 
     <ImageView
-        android:id="@+id/slide_Img"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:scaleType="centerCrop"
-        android:contentDescription="@string/slide_1_title"
-        android:src="@drawable/stax_slide_2"/>
+            android:id="@+id/slide_Img"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:contentDescription="@string/slide_1_title"
+            android:scaleType="centerCrop"
+            android:src="@drawable/stax_slide_2" />
 
     <TextView
-        android:id="@+id/slide_text_title"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:textSize="55sp"
-        android:textStyle="bold"
-        android:lineHeight="46dp"
-        android:fontFamily="font-bold"
-        android:layout_marginTop="@dimen/margin_34"
-        android:layout_marginHorizontal="@dimen/margin_24"
-        android:text="@string/slide_1_title"/>
+            android:id="@+id/slide_text_title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/margin_24"
+            android:layout_marginTop="@dimen/margin_34"
+            android:lineHeight="46dp"
+            android:text="@string/slide_1_title"
+            android:textSize="45sp"
+            app:fontFamily="@font/brutalista_bold" />
```

Comment:
Formatting is off plus remove the space here `app:fontFamily="@font/brutalista_bold"/>`